app/helper.js

```diff
@@ -134,7 +134,13 @@ class YoutubeGrabberHelper {
       publishedText = premiereDate.toLocaleString()
     } else if (typeof (video.viewCountText) === 'undefined') {
       premium = true
-      publishedText = video.publishedTimeText.simpleText
+      if(typeof (video.publishedTimeText) === 'undefined') {
+        var regex = new RegExp('^.*' + channelInfo.channelName + ' (.*?) ago.*', 'g')
+        publishedText = video.title.accessibility.accessibilityData.label.replace(regex, '$1')
+      }
+      else { // not sure if this is actually still needed
```

Comment:
Yes it is still required. On the same  channel are also premium videos with the publishedTimeText field available